src/Codeception/Command/Run.php

```diff
@@ -588,15 +584,15 @@ protected function matchTestFromFilename($filename, $testsPath)
             //codecept run tests
             return ['', '', $filter];
         }
-        $res = preg_match("~^$testsPath/(.*?)(?>/(.*))?$~", $filename, $matches);
+        $res = preg_match("~^{$testsPath}/(.*?)(?>/(.*))?$~", $filename, $matches);
```

Comment:
`#` as PREG delimiter in regex

Reply:
@hason Oops, you're right :)

This time I also searched for all [PCRE functions](https://www.php.net/manual/en/ref.pcre.php) and tried to catch them all.

Thanks for reviewing.